versions.json

```diff
@@ -33,6 +33,7 @@
     "cimg": "1.6.3",
     "cmake": "3.0.2",
     "cminpack": "1.3.4",
+    "coin": "3.1.3",
```

Comment:
I copied this from x264.mk, supposing there was a reason for this.  I guess it can be omitted.